internal/controller/standalone_pgadmin/controller.go

```diff
@@ -59,3 +116,343 @@ func (r *PGAdminReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		For(&v1beta1.PGAdmin{}).
 		Complete(r)
 }
+
+// reconcilePGAdminStatefulSet writes the StatefulSet that runs pgAdmin.
+func (r *PGAdminReconciler) reconcilePGAdminStatefulSet(
```

Comment:
I'm definitely interested in that idea. I kept things in the same file because that appears to be our normal practice when looking at our other controllers, but I like the idea of smaller files.

Reply:
i also could see a `controller.go` that deals with the applying changes - then individual files to "create" the resource :thinking: 

`controller.go` calls `pgadmin.Pod` functions that exist in a `pgadmin/pod.go` :thought_balloon:  then applies what it returns

so `internal/pgadmin/controller.go` has `reconcile*()` functions and `pgadmin/*.go` has `Pod()`, `ConfigMap()`, etc. functions